content/index.html.haml

```diff
@@ -67,6 +67,14 @@ homepage: true
   :image => {:src => expand_link('images/post-images/jenkins-is-the-way/jenkins-is-the-way.png'), :height => "285px"},
   :call_to_action => {:text => 'More info', :href => 'https://JenkinsIsTheWay.io/'}}
 
+// Ready to enable after DevOps World finishes
+// -# Hacktoberfest
```

Comment:
No need to wait IMHO. We can tear down the CDF Jumbotron if there is a concern about the number. We can also comment out the Participate one as an alternative